service/lib/ops/teamOps.js

```diff
@@ -332,71 +393,51 @@ var teamOps = {
   },
 
   /**
-   * Fetch specific team data
+   * Fetch the users data from a team
    *
-   * @param  {params}   params { id, organizationId }
+   * @param  {params}   params { id, page, limit }
    * @param  {Function} cb
    */
-  readTeam: function readTeam ({ id, organizationId }, cb) {
-    const tasks = []
-    let team
-
-    tasks.push((next) => {
-      const sql = SQL`
-        SELECT *
-        FROM teams
-        WHERE id = ${id}
-        AND org_id = ${organizationId}
-      `
-
-      db.query(sql, (err, result) => {
-        if (err) return next(Boom.badImplementation(err))
-        if (result.rowCount === 0) return next(Boom.notFound(`Team with id ${id} could not be found`))
+  readTeamUsers: function readTeamUsers ({ id, page, limit }, cb) {
+    const offset = (page < 1 ? 1 : page) * limit - limit
+    const job = {
+      id: id,
+      offset: offset,
+      limit: limit < 0 ? 0 : limit,
+      team: {}
+    }
 
-        team = mapping.team(result.rows[0])
+    loadTeamUsers(job, (err) => {
+      if (err) return cb(err)
 
-        team.users = []
-        team.policies = []
-        next()
-      })
+      return cb(null, job.team.users)
```

Comment:
Would be useful to return some more meta-data here, i.e current page, total users etc?